### PR TITLE
Enable OAuth2 on module. 

### PR DESCRIPTION
**Pull request by thomasegense**
Enable OAuth2 on module. No service methods are so far defined to require a token.

### CODE REVIEW

File: src/main/openapi/ds-storage-openapi_v1.yaml [hunk 1]
```diff
@@ -817,5 +862,29 @@ components:
...
+  # TODO: Inject this from properties when DRA-139 has been implemented        
+  securitySchemes:
+
+    KBOAuth:
+      type: oauth2
+      description: 'This API uses OAuth 2 with the implicit grant flow. [More info](https://api.example.com/docs/auth)'
+      flows:
+        implicit:   # <---- OAuth flow(authorizationCode, implicit, password or clientCredentials)
+          authorizationUrl: 'https://keycloak-devel-01.kb.dk/realms/ds-poc/protocol/openid-connect/auth'
+          #authorizationUrl: https://api.example.com/oauth2/authorize # Overridden by the configuration
```
Comment (VictorHarbo) [inline src/main/openapi/ds-storage-openapi_v1.yaml:876]: As in present and datahandler, please inject this with ${config.yaml.path}

**Review by VictorHarbo — CHANGES_REQUESTED** [review]
Looks good. However, it seems like the version deployed to devel has a method locked behind Oauth. Integration test is failing due to not being authorized to access [/record/{id} endpoint. My guess is you've been testing with this one. Please deploy updated version for a final check. Also one comment in code.

File: src/main/openapi/ds-storage-openapi_v1.yaml (hunk 1 above)
Comment (thomasegense) [reply src/main/openapi/ds-storage-openapi_v1.yaml:876]: fixed

**Review by VictorHarbo — APPROVED** [review]
Looks good now

**Pull request merged**